### ISSUE TITLE
Bug: Filament version 3.2.23 breaks this package

### ISSUE THREAD

**Issue by maartenpaauw** [cheesegrits/filament-google-maps#76]
In version 3.2.23 of Filament a new trait `HasWidth` is introduced and applied to the base `Column` class. The types of the `width` attribute, `width(...)` and `getWidth(...)` methods mismatch.

See pull request: https://github.com/filamentphp/filament/pull/11148

This will result in the following error:

```php
Symfony\Component\ErrorHandler\Error\FatalError
Type of Cheesegrits\FilamentGoogleMaps\Columns\MapColumn::$width must be Closure|string|int|null (as in class Filament\Tables\Columns\Column)
```

**Comment by cheesegrits** [comment]
Thanks for the report, apologies for the delayed response, I have some health issues going on.

Working on it now.

**Comment by maartenpaauw** [comment]
Closing this issue because it is solved in version 3.0.13: https://github.com/cheesegrits/filament-google-maps/releases/tag/v3.0.13
